pkgs/by-name/ti/tic-80/package.nix

```diff
@@ -0,0 +1,143 @@
+{ lib
+, pkgs
+, stdenv
+, fetchFromGitHub
+, libGL
+, alsa-lib
+, libX11
+, xorgproto
+, libICE
+, libXi
+, libXScrnSaver
+, libXcursor
+, libXinerama
+, libXext
+, libXxf86vm
+, libXrandr
+, libxkbcommon
+, wayland
+, wayland-protocols
+, wayland-scanner
+, dbus
+, udev
+, libdecor
+, pipewire
+, libpulseaudio
+, libiconv
+, ...
+}:
+with pkgs;
+stdenv.mkDerivation rec {
+  pname = "tic-80";
+  version = "1.1.2837";
+
+  src = fetchFromGitHub {
+    owner = "nesbox";
+    repo = "TIC-80";
+    rev = "v" + version;
+    sha256 = "sha256-2hSaLWw57F19tSfgJvBgbqW52vCu8p/TmoybYzQEybE=";
+    fetchSubmodules = true;
+    leaveDotGit = true; # Required for TIC-80's version detection via CMake.
+  };
+
+  # To avoid the awkward copyright range of "2017-1980", which would be caused by the
+  # sandbox environment, hardcode the year of the release.
+  postUnpack = ''
+    YEAR_OF_LAST_COMMIT=$(git -C source log -1 --format=%ad --date=format:%Y)
```

Comment:
Can we hardcode this to a known good year? We really do not want to rely on .git directory.